src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -324,7 +324,9 @@ namespace OpenRCT2::Ui::Windows
                     }
                 }
 
-                currentRide->setToDefaultInspectionInterval();
+                // Do not reset inspection interval when closing the construction window.
+                // The player may have changed this in the ride window (see issue #25601).
+                // currentRide->setToDefaultInspectionInterval();
```

Comment:
Aside from this not being the solution (see review), there is no point commenting out incorrect code and annotating it. Just remove it. We have Git in case we ever need it back or refer to it.